src/status_im2/contexts/wallet/create_account/edit_derivation_path/component_spec.cljs

```diff
@@ -0,0 +1,20 @@
+(ns status-im2.contexts.wallet.create-account.edit-derivation-path.component-spec
+  (:require
+    [status-im2.contexts.wallet.create-account.edit-derivation-path.view :as edit-derivation-path]
+    [test-helpers.component :as h]))
+
+(h/describe "Render"
```

Comment:
Nice! Maybe you could add a clearer description, e.g "edit derivation path page"

Reply:
Fixed